models/public/wavernn/description/wavernn.md

```diff
@@ -0,0 +1,134 @@
+# wavernn
+
+## Use Case and High-Level Description
+
+WaveRNN is a model for the text-to-speech task originally trained in PyTorch\*
+then converted to ONNX\* format.
+WaveRNN performs waveform regression from mel-spectrogram.
+For details see [paper](https://arxiv.org/pdf/1703.10135.pdf), [repository](https://github.com/as-ideas/ForwardTacotron).
+
+## ONNX Models
+We provide pretrained models in ONNX format for user convenience.
+
+### Steps to Reproduce PyTorch to ONNX Conversion
+Model is provided in ONNX format, which was obtained by the following steps.
+
+1. Clone the original repository
+```sh
+git clone https://github.com/as-ideas/ForwardTacotron
+cd ForwardTacotron
+```
+2. Checkout the commit that the conversion was tested on:
+```sh
+git checkout 78789c1aa845057bb2f799e702b1be76bf7defd0
+```
+4. Copy provided script `wavernn_to_onnx.py` to ForwardTacotron root directory, and apply git path `0001-Added-batch-norm-fusing-to-conv-layers.patch`.
+5. Download WaveRNN model from https://github.com/fatchord/WaveRNN/tree/master/pretrained/ and extract in to pretrained directory.
+```sh
+mkdir pretrained
+wget https://raw.githubusercontent.com/fatchord/WaveRNN/master/pretrained/ljspeech.wavernn.mol.800k.zip
+unzip ljspeech.wavernn.mol.800k.zip -d pretrained && mv pretrained/latest_weights.pyt pretrained/wave_800K.pyt
+```
+6. Run provided script for conversion WaveRNN to onnx format
+```sh
+python3 wavernn_to_onnx.py --mel <path_to_some_mel_spectrogram_file_from_train_directory>.npy --voc_weights pretrained/wave_800K.pyt --hp_file hparams.py --force_cpu --batched
```

Comment:
1. There is no `train` directory.
2. Even if there was, you didn't just obtain the models by using _some_ spectrogram file. You used a specific file. Please specify what it was.
3. There is no `--force_cpu` option.